admin/routes/index.js

```diff
@@ -94,7 +93,7 @@ function getBuildNumber () {
   })
 }
 
-router.get('/ping', (req, res) => getPing(req, res))
+router.get('/ping', (req, res) => res.status(200).send('OK'))
```

Comment:
Still doesn't look right... should return the commit and and buildId

Reply:
This has to be a bad merge... I've fixed it.